roles/openshift-applier/tasks/process-content.yml

```diff
@@ -6,8 +6,11 @@
   - "{{ entry.content | default([]) }}"
 
 - name: "Apply OpenShift Cluster Content ('{{ entry.object }}') - based on directory"
-  command: >
-    oc apply -f {{ tmp_inv_dir }}{{ entry.content_dir }}
+  block:
+  - fail:
+      msg: "This functionality is being deprecated. Please use the 'content.file' variable instead. See role docs for more info."
```

Comment:
@etsauer unless you include `ignore_errors: True` here, it will fail ... we probably don't want to fail. Also, could we start the message with an all-caps `WARNING: This functionality ....`